package.json

```diff
@@ -21,15 +21,15 @@
     "node-gyp-build": "^4.8.0"
   },
   "peerDependencies": {
-    "tree-sitter": "^0.21.0"
+    "tree-sitter": "^0.22.4"
```

Comment:
This seems odd/unusual... Why is the library dependency version different from the CLI version below?